e2e/internal/client/cli.go

```diff
@@ -252,7 +252,7 @@ func NewCLI() (*CLI, error) {
 	// your file system and docker image repo isn't polluted
 	// with test data and files. Since this is going to run
 	// from Docker, the binary will be located in the root bin.
-	cliPath := filepath.Join("/", "bin", "copilot")
+	cliPath := filepath.Join("/Users/penghaoh/Work/PRIVATE-amazon-ecs-archer/bin/local/copilot")
```

Comment:
I'm a little confused. This new `cliPath` will apply to all of the e2e tests, right? Is this what we want?